src/status_im2/contexts/wallet/send/select_address/view.cljs

```diff
@@ -41,70 +45,144 @@
-        [quo/address-input
-         {:on-scan               #(rf/dispatch [:open-modal :scan-address])
-          :ens-regex             constants/regx-ens
-          :address-regex         constants/regx-address
-          :scanned-value         scanned-address
-          :on-detect-ens         on-detect-address-or-ens
-          :on-detect-address     on-detect-address-or-ens
-          :on-change-text        (fn [text]
+  [input-value input-focused?]
+  (fn []
+    (let [scanned-address       (rf/sub [:wallet/scanned-address])
+          valid-ens-or-address? (boolean (rf/sub [:wallet/valid-ens-or-address?]))]
+      [quo/address-input
+       {:on-focus              #(reset! input-focused? true)
+        :on-blur               #(reset! input-focused? false)
+        :on-scan               #(rf/dispatch [:open-modal :scan-address])
+        :ens-regex             constants/regx-ens
+        :address-regex         constants/regx-address
+        :scanned-value         scanned-address
+        :on-detect-ens         #(debounce/debounce-and-dispatch
+                                 [:wallet/validate-ens %]
+                                 300)
+        :on-detect-address     #(debounce/debounce-and-dispatch
+                                 [:wallet/validate-address %]
+                                 300)
+        :on-change-text        (fn [text]
+                                 (let [starts-like-eth-address (re-matches
+                                                                constants/regx-address-fragment
+                                                                text)]
                                    (when-not (= scanned-address text)
                                      (rf/dispatch [:wallet/clean-scanned-address]))
-                                   (reset! input-value text))
-          :on-clear              #(rf/dispatch [:wallet/clean-scanned-address])
-          :valid-ens-or-address? @valid-ens-or-address?}]))))
+                                   (if starts-like-eth-address
+                                     (rf/dispatch [:wallet/fetch-address-suggestions text])
+                                     (rf/dispatch [:wallet/clean-local-suggestions]))
+                                   (reset! input-value text)))
+        :valid-ens-or-address? valid-ens-or-address?}])))
+
+(defn- ens-linked-address
+  [{:keys [address networks theme]}]
+  [quo/text
+   {:size  :paragraph-2
+    :style {:padding-horizontal 12
+            :padding-top        4}}
+   (map (fn [network]
+          ^{:key (str network)}
+          [quo/text
+           {:size  :paragraph-2
+            :style {:color (colors/resolve-color network theme)}}
+           (str (subs (name network) 0 3) ":")])
+        networks)
+   [quo/text
+    {:size   :paragraph-2
+     :weight :monospace
+     :style  {:color (colors/theme-colors colors/neutral-100 colors/white theme)}}
+    (string/replace address "x" "×")]])
```

Comment:
I see in another file you replaced this line `(string/replace address "x" "×")` with https://github.com/status-im/status-mobile/pull/17579/files#diff-2e71df9762267904884e9f24b17ba52a4b7c6db32424414daa97edbf59f3bf80L36-R37 Is this required?

Reply:
Btw I am currently working on implementing a network text component to fix these problems and avoid the repetition https://github.com/status-im/status-mobile/issues/17628

Reply:
I think `(string/replace address "x" "×")` is not the right way to do it. The letter X should automatically be written as `×` when using monospace. I think the font needs to be fixed. We also need a bold variant of that font for some designs.

Reply:
Yep, it was just a temp approach, but actually it doesn't make sense if we are going to fix it when we update the font(s). I'll remove it.